EcoSoapBank/Helpers/UIKit/UIViews+Convenience.swift

```diff
@@ -27,10 +27,25 @@ extension UILabel {
 
 extension UIImageView {
     /// Initializes a UIImageView with image name
-    /// - Parameter imageNamed: **String** with the system name for the image
-    convenience init(_ imageNamed: String) {
-        self.init(image: UIImage(systemName: imageNamed, withConfiguration: UIImage.SymbolConfiguration(pointSize: 16, weight: .semibold)))
+    /// - Parameter systemName: **String** with the system name for the image
+    convenience init(_ systemName: String) {
+        self.init(image: UIImage(systemName: systemName, withConfiguration: UIImage.SymbolConfiguration(pointSize: 16, weight: .semibold)))
         translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Also seems odd to not pass in a configuration in the init for the view, yet to specify one for the image. Maybe add withConfiguration for the imageView's init and give it this default value. Also, should note that it sets the translatesAutoResizingMaskIntoConstraints in the docs for this function so it is clear when looking at quick docs.